.nextmv/update_apps.py

```diff
@@ -126,20 +141,59 @@ def get_manifest(
 ) -> dict[str, Any]:
     """Returns the manifest from the S3 bucket."""
 
-    result = client.get_object(Bucket=bucket, Key=f"{folder}/{manifest_file}")
-    return yaml.safe_load(result["Body"].read())
+    log("Getting manifest from S3 bucket.")
 
+    manifest = {}
+    try:
+        result = client.get_object(Bucket=bucket, Key=f"{folder}/{manifest_file}")
+        manifest = yaml.safe_load(result["Body"].read())
+        log("Obtained manifest from S3 bucket.")
 
-def tar_app(name: str, version: str) -> str:
-    """Create a tarbal of the app. Returns the name of the tarball."""
+    except client.exceptions.NoSuchKey:
+        log("No manifest in S3 bucket.")
```

Comment:
could we add the bucket we and key we were looking at for debugging purposes?

Reply:
They are configured as secrets on purpose 😸. Even if I printed them I would expect GitHub to obfuscate them.

Reply:
yeah, was apparently too tired. did not see they were secrets.